src/vm/reflectioninvocation.cpp

```diff
@@ -760,6 +760,15 @@ class ArgIteratorBaseForMethodInvoke
         LIMITED_METHOD_CONTRACT;
     }
 
+#ifdef UNIX_AMD64_ABI
```

Comment:
I don't see the need for the `UNIX_AMD_ABI` since this is the default case for use.

Reply:
The `MethodDesc::IsRegPassedStruct` (which this uses) is `#ifdef`'d in for only `UNIX_AMD64_ABI`. Additionally, this logic is only used when `UNIX_AMD64_ABI` is defined anyway (since that's the only time that it is relevant.

Reply:
>Additionally, this logic is only used when UNIX_AMD64_ABI is defined anyway (since that's the only time that it is relevant.

Usage is fine to be `#ifdef` out, but definitions largely have no reason to be defined out _unless_ they won't compile or will lie and in this case neither should be an issue. They will compile just fine and will return the truth. I would prefer to use `#ifdef`s sparingly.